src/main/java/org/broadinstitute/dsm/jobs/UPSTrackingJob.java

```diff
@@ -195,25 +188,26 @@ private static void updateTrackingInfo(String statusType, String oldType, String
                 if (shouldTriggerEventForKitOnItsWayToParticipant(statusType, oldType)) {
                     KitDDPNotification kitDDPNotification = KitDDPNotification.getKitDDPNotification(SQL_SELECT_KIT_FOR_NOTIFICATION_EXTERNAL_SHIPPER + SELECT_BY_TRACKING_NUMBER, new String[] { DELIVERED, trackingId }, 2);//todo change this to the number of subkits but for now 2 for test boston works
                     if (kitDDPNotification != null) {
-                        logger.info("Triggering DDP for delivered kit with external order number: " + kit.getExternalOrderNumber());
+                        logger.info("Triggering DDP for kit going to participant with external order number: " + kit.getExternalOrderNumber());
                         EventUtil.triggerDDP(kitDDPNotification);
-
                     }
                     else {
                         logger.error("delivered kitDDPNotification was null for " + kit.getExternalOrderNumber());
                     }
                 }
 
             }
-            else {
-                //if picked up place order
-                if (shouldMakeCEOrder(statusType) && !kit.isCEOrdered()) {
-                    Instant now = Instant.now();
```

Comment:
This `Instant.now()` under the right circumstances is usually fairly close to the shipping time, but it's not precise.  Depending on how much time has elapsed between this method's awareness of a kit coming back and the actual dates, the collection date used via `Instant.now()` can be considerably far from reality.